lib/rubocop/cop/naming/rescued_exceptions_variables_name.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Naming
+      # This cop makes sure that rescued exceptions variables are named
+      # exception.
+      #
+      # @example
+      #   # bad
+      #   begin
+      #     # do something
+      #   rescue MyException => error
+      #     # do something
+      #   end
+      #
+      #   # good
+      #   begin
+      #     # do something
+      #   rescue MyException => exception
+      #     # do something
+      #   end
+      #
+      class RescuedExceptionsVariablesName < Cop
+        MSG = 'Use `exception` instead of `%<bad>s`.'.freeze
+
+        def on_resbody(node)
+          exception_type, exception_name = node.children
```

Comment:
Btw, it's best to use node patters instead of manual decomposing nodes. We doing decomposition we prefer to use `*node` over `node.children`.